Tools/TransformTool/TransformTool.cs

```diff
@@ -737,7 +732,24 @@ void UpdateManipulatorAction()
 
         public bool IsTwoHandedScaling(Transform rayOrigin)
         {
-            return m_Scaling && m_GrabData.Any(kvp => kvp.Value.rayOrigin == rayOrigin);
+            return m_Scaling && GrabDataForRayOrigin(rayOrigin) != null;
+        }
+
+        GrabData GrabDataForNode(Node node)
+        {
+            return node == Node.LeftHand ? m_LeftGrabData : m_RightGrabData;
+        }
+
+        GrabData GrabDataForRayOrigin(Transform rayOrigin)
+        {
+            var grabData = m_LeftGrabData;
+            if (grabData != null && grabData.rayOrigin == rayOrigin)
```

Comment:
Is this correct?  If the  leftGrabData rayOrigin is the rayOrigin, we return the rightGrabData?

Reply:
Good catch. This was a refactor gone bad.

P.S. @stella3d sorry if you had already committed on top of the current state. It should auto-merge just fine if you pull my changes, then you should be able to push.